library/models/library.py

```diff
@@ -185,9 +185,9 @@ def onchange_day_to_return_book(self):
         @return : Dictionary having identifier of the record as key
                   and the book return date as value'''
         t = "%Y-%m-%d %H:%M:%S"
-        rd = relativedelta(days=self.day_to_return_book or 0.0)
-        if self.date_issue and rd:
-            ret_date = datetime.strptime(self.date_issue, t) + rd
+        diff = rd(days=self.day_to_return_book or 0.0)
```

Comment:
is rd() a function or you made a shortcut?

Reply:
Here rd is short name given to relativedelta which is imported.  from dateutil.relativedelta import relativedelta as rd